Question:
What does the following error message tell you?

ArgumentError: wrong number of arguments (1 for 2)
  from (irb):1:in `calculate_product'
  from (irb):4
  from /Users/username/.rvm/rubies/ruby-2.5.3/bin/irb:12:in `<main>'
  
Answer:
The error message tells us that the method invocation calculate_product needs to
pass two arguements, but we only pass one.
